docs/UserGuide.md

```diff
@@ -587,72 +629,82 @@ Format: `clear`
-Find           | `find KEYWORDS` <br> e.g. `find John Kite`
-List           | `list` 
-Delete         | `delete INDEX` <br> e.g. `delete 3`
-Clear          | `clear`
+Command        | Format        | Example
+---------------|--------------|-----------------------------------------------------------------------------------
+Add            | `add n/NAME r/ROLE e/EMAIL [p/PHONE] [d/DESCRIPTION]`   |   `add n/Johnny r/stk e/johnny@example.com p/12345678 d/Business Analyst`
+Edit           | `edit INDEX [n/NAME] [r/ROLE] [e/EMAIL] [p/PHONE] [d/DESCRIPTION]`   |   `edit 1 n/John r/dev e/john@email.com`
+View           | `view INDEX`   |   `view 2`
+Find           | `find KEYWORDS`   |   `find John Kite`
+List           | `list`   | `list`
+Delete         | `delete INDEX`   |   `delete 3`
+Clear          | `clear`  | `clear`
 
 ---------------------------------------------------------------------------------------------------------------------
 
 ## Glossary
 
-\# |            Term                | Description
----|--------------------------------|------------------------------------------------------------------------------------------------|
-1  | Command Line Interface (CLI)   | A text-based user interface (UI) used to view and manage computer files.                       |
-2  | Graphical User Interface (GUI) | A system of interactive visual components for computer software.                               |
-3  | Deliverable                    | An item to be completed as part of the product development process.                            |
-4  | Milestone                      | A stage in the software development process associated with a particular group of deliverables.|
-5  | Mode                           | The state of the application that affects how each command will be executed. The app can be in dashboard, deliverable, meeting or contact mode.                                                  |
-6  | Stakeholder                    | An external party involved with the product.                                                   |
+Term                | Description
+--------------------------------|------------------------------------------------------------------------------------------------|
+Command Line Interface (CLI)   | A text-based user interface (UI) used to view and manage computer files.                       |
+Graphical User Interface (GUI) | A system of interactive visual components for computer software.                               |                         |
+Milestone                      | A stage in the software development process associated with a particular group of deliverables. It is a non-negative integer, or a period-separated string of non-negative integers. |
+Mode                           | The state of the application that affects how each command will be executed. The app can be in dashboard, deliverable, meeting or contact mode.                                                  |
```

Comment:
Able to include OCP too? i.e. "Overall Completion Percentage (OCP) | A donut chart showing the product’s completion status, found on the left panel of the dashboard mode."

modified from DG.